backend/main.py

```diff
@@ -113,6 +113,36 @@ def new_game():
   return jsonify(firebase.put('/games', game, put_data))
 
 
+@app.route('/joingame', methods=['POST'])
+def join_game():
+  request_data = request.get_json()
+  game_to_join = request_data['gameId']
+  player_to_add_to_game = request_data['playerId']
+  name = request_data.get('name', '')
+  need_gun = request_data.get('needGun', False)
+  profile_image_url = request_data.get('profileImageUrl', '')
+  start_as_zombie = request_data.get('startAsZombie', False)
+  volunteer = request_data.get('volunteer', False)
+  be_secret_zombie = request_data.get('beSecretZombie', False)
+  user_id = request.headers['userId']
+
+  player_info = {
+    'gameId': game_to_join
+  }
+  firebase.put('/users/' + user_id + '/players', player_to_add_to_game, player_info)
```

Comment:
Sorry I didn't make this clearer in the doc. The name was chosen because when a User joins a Game, it results in a Player. If you consider it from the User's perspective, the Player is the result.

Wish there was a more elegant way to say "userJoinGame" or something, it's more clear, but also awkward.

addPlayer also works, as does createPlayer.